components/text/index.html

```diff
@@ -0,0 +1,40 @@
+<html>
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" type="text/css" href="../css/styles.css" />
+    <title>Text</title>
+  </head>
+  <body>
+    <!-- HTML for headers -->
+
+    <h1>Header Text - h1</h1>
+    <h2>Header Text - h2</h2>
+    <h3>Header Text - h3</h3>
+    <h4>Header Text - h4</h4>
+    <h5>Header Text - h5</h5>
+    <br />
```

Comment:
better to not use br tag i thing

Reply:
Yeah I will remove it while adding documentation and code snippets.